### PR TITLE
Consolidate boilerplate in integration tests

### PR DESCRIPTION
**Pull request by ethantkoenig**
Move error-checking and other shared boilerplate to helper functions; makes tests less verbose.

### CODE REVIEW

**Comment by andreynering** [comment]
LGTM

**Comment by pgaskin** [comment]
LGTM

File: integrations/integration_test.go [hunk 1]
```diff
@@ -155,21 +156,19 @@ func (s *TestSession) MakeRequest(t *testing.T, req *http.Request) *TestResponse
...
-	req = NewRequestBody(t, "POST", "/user/login",
-		bytes.NewBufferString(url.Values{
-			"_csrf":     []string{doc.GetInputValueByName("_csrf")},
-			"user_name": []string{userName},
-			"password":  []string{password},
-		}.Encode()),
-	)
+	doc := NewHtmlParser(t, resp.Body)
+	req = NewRequestWithValues(t, "POST", "/user/login", map[string]string{
+		"_csrf":     doc.GetInputValueByName("_csrf"),
```
Comment (lafriks) [inline integrations/integration_test.go]: This should be changed to `GetCSRF`

Comment (ethantkoenig) [reply integrations/integration_test.go]: Good catch!

File: integrations/integration_test.go [hunk 2]
```diff
@@ -155,21 +156,19 @@ func (s *TestSession) MakeRequest(t *testing.T, req *http.Request) *TestResponse
 	return resp
 }
 
-func loginUser(t *testing.T, userName, password string) *TestSession {
```
Comment (lunny) [inline integrations/integration_test.go:158]: No, I don't think we should do that. How should we test user login with different password?

Comment (ethantkoenig) [reply integrations/integration_test.go:158]: How about having two functions?

```
func loginUserDefaultPassword(...) {
	loginUser(..., "password")
}

func loginUser(...) {
	// what we originally had
}
```

**Comment by appleboy** [comment]
@ethantkoenig Could you rebase the master? something wrong with drone sig file.

**Comment by ethantkoenig** [comment]
@appleboy Rebased

File: integrations/integration_test.go (hunk 2 above)
Comment (ethantkoenig) [reply integrations/integration_test.go:158]: @lunny Done

**Comment by lafriks** [comment]
LGTM

**Pull request merged**